addons/mail/static/tests/chatter/web/chatter.test.js

```diff
@@ -577,6 +576,33 @@ test("chatter updating", async () => {
     await contains(".o-mail-Message");
 });
 
+test("chatter message actions appear only after saving the form", async () => {
+    const pyEnv = await startServer();
+    const partnerId = pyEnv["res.partner"].create(
+        { display_name: "first partner" },
+    );
+    pyEnv["mail.message"].create({
+        body: "Auto message",
+        model: "res.partner",
+        res_id: partnerId,
+    });
+    await start();
+    await openFormView("res.partner", partnerId, {
+        arch: `
+            <form string="Partners">
+                <sheet>
+                    <field name="name"/>
+                </sheet>
+                <chatter/>
+            </form>`,
+        resIds: [partnerId]
+    });
+    await contains(".o_field_widget[name='name']");
+    expect(".o-mail-Message-actions").not.toBeDisplayed();
```

Comment:
Better
```js
await contains(".o-mail-Message");
await contains(".o-mail-Message-actions", { count: 0 });
```

This gives more confidence the awaiting for message rendering is the same as actions so asserting no actions is good.
Note that chatter rendering differs from form view hence `await field[name]` is not a good enough condition to negatively assert chatter is loaded for negative assertion.